src/components/application_manager/src/app_launch/app_launch_data_json.cc

```diff
@@ -241,8 +260,8 @@ bool AppLaunchDataJson::DeleteOldestAppData() {
 
   // Copy in temporary vector Json list without oldest record
   int32_t i = 0;
-  for (Json::Value::iterator it = GetSavedApplicationDataList().begin();
-       it != GetSavedApplicationDataList().end();
+  apps_list = GetSavedApplicationDataList(dictionary);
```

Comment:
@LitvinenkoIra Why did you call this function again?

Reply:
@AByzhynar removed 86d7ed7ca314f780ee977f843c0bacff8b471c4a